lib/Language/Haskell/Homplexity/Utilities.hs

```diff
@@ -1,12 +1,22 @@
 module Language.Haskell.Homplexity.Utilities(
     sumOf
   , maxOf
+  , declHeadName
   ) where
 
+import Language.Haskell.Exts.Syntax (DeclHead(..), Name)
+
 -- | Maximum of the results of mapping the function over the list.
 maxOf :: (a -> Int) -> [a] -> Int
 maxOf f = maximum . (0:). map f
 
 -- | Sum the results of mapping the function over the list.
 sumOf :: (a -> Int) -> [a] -> Int
 sumOf f = sum . map f
+
+-- | Get the name of a declaration.
```

Comment:
`declHeadName` is only used once, so not sure we need it in another module?

Reply:
It seems to me that it could be reused anytime we want to analyze declarations. For example type class instances.